code/modules/power/singularity/boh_tear.dm

```diff
@@ -16,13 +16,19 @@
 	allowed_size = STAGE_SIX
 	var/ghosts = list()
 	var/old_loc
+	var/start_time = 0
 
 /obj/singularity/boh_tear/Initialize()
 	. = ..()
 	old_loc = loc
+	start_time = world.time
 	QDEL_IN(src, 5 SECONDS) // vanishes after 5 seconds
 
 /obj/singularity/boh_tear/process()
+	//Backup to catch timerss errors
+	if(start_time + (10 SECONDS) < world.time)
+		stack_trace("The timer subsytem isn't firing properly, yell at your local coders posthaste")
+		qdel(src)
```

Comment:
You test this error? Or it can occurred only on loaded server?
Recently i had problems with disabling of processing.

Reply:
I haven't personally tested it. The current method is intended to catch a failed timerss. I assume that this is due to lag, there's a chance it's caused by goof's explosion optimizations.

Reply:
Can't seem to recreate in testing, I assume it's an issue with lag.

Reply:
my explosion optimizations aren't merged rn due to issues